src/i-output.ts

```diff
@@ -0,0 +1,118 @@
+/**
+ * Interface for output state.
+ */
+export type IOutput = any;
+
+/*
+NBFORMAT SCHEMA
```

Comment:
Probably in addition to, since users do fun things and not all bugs are caught during compilation. 😉